nyc/bills.py

```diff
@@ -1,232 +1,359 @@
-from legistar.bills import LegistarBillScraper
+from legistar.bills import LegistarAPIBillScraper
 from pupa.scrape import Bill, VoteEvent
 from pupa.utils import _make_pseudo_id
 import datetime
 from collections import defaultdict
 import pytz
 
-class NYCBillScraper(LegistarBillScraper):
+from .secrets import TOKEN
+
+class NYCBillScraper(LegistarAPIBillScraper):
     LEGISLATION_URL = 'http://legistar.council.nyc.gov/Legislation.aspx'
-    BASE_URL = 'http://legistar.council.nyc.gov/'
-    TIMEZONE = "US/Eastern"
+    BASE_URL = 'https://webapi.legistar.com/v1/nyc'
+    BASE_WEB_URL = 'http://legistar.council.nyc.gov'
+    TIMEZONE = 'US/Eastern'
 
-    VOTE_OPTIONS = {'affirmative' : 'yes',
-                    'negative' : 'no',
-                    'conflict' : 'absent',
+    VOTE_OPTIONS = {'affirmative': 'yes',
+                    'negative': 'no',
+                    'conflict': 'absent',
                     'maternity': 'excused',
-                    'paternity' : 'excused',
+                    'paternity': 'excused',
                     'bereavement': 'excused',
-                    'non-voting' : 'not voting',
-                    'jury duty' : 'excused',
-                    'absent' : 'absent',
-                    'medical' : 'excused'}
+                    'non-voting': 'not voting',
+                    'jury duty': 'excused',
+                    'absent': 'absent',
+                    'medical': 'excused'}
+
+    # TO-DO: cq legislative session including 1996
+    SESSION_STARTS = (2014, 2010, 2006, 2004, 2002, 1998, 1996)
```

Comment:
Why do these not all have the same gap between them?

Reply:
Good question. Except 1996, which is a placeholder until we learn when the legislative session including 1996 began, these years are sourced from from Legistar.

<img width="521" alt="screen shot 2017-11-06 at 11 40 31 am" src="https://user-images.githubusercontent.com/12176173/32455264-54c76108-c2e7-11e7-84cf-df0e3df5353f.png">

[I have an issue seeking clarification open over in **nyc-council-councilmatic**](https://github.com/datamade/nyc-council-councilmatic/issues/4).